_Tests/AudibleApi.Tests/L0/ApiTests.Download.cs

```diff
@@ -322,10 +336,14 @@ public async Task download_success()
 			var badExt = ".xyz";
 			var goodExt = ".aax";
 
+			var asin = "B07L162HDY";
```

Comment:
I meant to mention earlier: if you have any popular books, feel free to use their asin.s instead of the ones I used. The one here was an odd choice by me but others I chose because I thought it would be more likely that future people would already have them. eg: Sherlock Holmes, Harry Potter. There's another I sometimes use (Mimi's Adventure : B079DZ8YMP) specifically because it's a tiny book -- 3 minutes. This can also be replaced by another popular short title.